src/enforester.js

```diff
@@ -63,7 +63,7 @@ const EXPR_LOOP_OPERATOR = {};
 const EXPR_LOOP_NO_CHANGE = {};
 const EXPR_LOOP_EXPANSION = {};
 
-function getLineNumber(x: Syntax | T.Term) {
+function getLineNumber(x: any) {
```

Comment:
On second thought, unless you have any objections I'll do the refactoring in a separate PR. I realized while trying to put the types back in that the right way to get flow to pass essentially requires syntax/term unification #632. I'll work on that next but probably worth separating it out.